cinema/urls.py

```diff
@@ -1 +1,23 @@
-# write urls here
+from django.urls import path, include
+from rest_framework import routers
+
+from cinema.views import (
+    ActorViewSet,
+    GenreViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+    CinemaHallViewSet
+)
+
+router = routers.DefaultRouter()
+router.register("genres", GenreViewSet)
+router.register("actors", ActorViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+
+urlpatterns = [
+    path("", include(router.urls))
+]
```

Comment:
If a basepath isn't specified, there's no need to include the router. Simply utilize `urlpatterns = router.urls`